Elements/src/Math/Domain1d.cs

```diff
@@ -124,6 +132,16 @@ public Domain1d[] DivideByCount(int n)
             return results;
         }
 
-
+        /// <summary>
+        /// Calculate the middle of the domain.
+        /// </summary>
+        public double Mid()
+        {
```

Comment:
I think this code is wrong in the non-increasing case. Can't this just be `(this.Min + this.Max) / 2` in all cases?

Reply:
Done.